src/app/core/custom-material/custom-material.module.ts

```diff
@@ -23,7 +24,10 @@ import {
         MatTooltipModule,
         MatGridListModule,
         MatRadioModule,
-        MatSnackBarModule
+        MatSnackBarModule,
+        MatDialogModule,
+        MatDatepickerModule,
+        MatNativeDateModule
```

Comment:
Please refer to this [documentation](https://material.angular.io/components/datepicker/overview#choosing-a-date-implementation-and-date-format-settings) to see why is it required and what are the other alternatives